demisto_sdk/commands/lint/lint_manager.py

```diff
@@ -565,16 +612,22 @@ def report_summary(self, lint_status: dict):
                                                  subsequent_indent=' ' * len(fail_pack_prefix))
         # intersection of all failed packages
         failed: Set[str] = set()
-        for packs in lint_status.values():
-            failed = failed.union(packs)
+        warnings: Set[str] = set()
+        for key in lint_status:
+            if key.startswith('fail'):
+                failed = failed.union(lint_status[key])
+            if key.startswith('warning'):
```

Comment:
If it failed, it is ok that this apck has warnings, but we want to report it as a failure. right?
```suggestion
           elif key.startswith('warning'):
```

Reply:
yes , but we still want to show its warnings too, if there are

Reply:
ok, let's make sure we male it clear.
e.g - passing packs with warnings, and failed packs with warnings.